frame.go

```diff
@@ -185,6 +186,16 @@ func (f *Frame) Service() string {
 	return string(f.Payload[_serviceNameIndex : _serviceNameIndex+l])
 }
 
+// IsLast indicates whether the frame has a continuation or streaming bit set.
+func (f *Frame) IsLast() bool {
+	if mt := f.messageType(); mt != messageTypeCallReq && mt != messageTypeCallReqContinue && mt != messageTypeCallRes && mt != messageTypeCallResContinue {
+		// This message type can't be continued or streamed.
+		return false
+	}
+	flags := f.Payload[_flagsIndex]
+	return (flags == hasMoreFragmentsFlag || flags == isStreamingFlag)
```

Comment:
i think we only care about the `hasMoreFragmentsFlag`. The `streaming` flag is only ever set on call req/call res to say different timeout semantics apply, but streaming calls have to correctly specify `hasMoreFragmentsFlag`

You want to do a bitwise check here too, not just check ` == `
```go
return flags & hasMoreFragmentsFlag != 0
```